Eastern-Research-Group/expert-query
# Feature/277 check 508

## Related Issues:
* [EQ-277](https://jira.epa.gov/browse/EQ-277)

## Main Changes:
* Updated several contrast and WAI-ARIA items in response to checking the UI with various accessibility tools.

## Steps To Test:
1. Check the changes with automated tools (or review the materials supplied externally) to verify major issues have been resolved.
